modules/core/src/main/java/org/locationtech/jts/index/strtree/BoundablePairDistanceComparator.java

```diff
@@ -0,0 +1,53 @@
+package org.locationtech.jts.index.strtree;
```

Comment:
New files require a license header.

The form of the header is...

/*
 * Copyright (c) 2016 Vivid Solutions.
 *
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Public License v1.0
 * and Eclipse Distribution License v. 1.0 which accompanies this distribution.
 * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
 * and the Eclipse Distribution License is available at
 *
 * http://www.eclipse.org/org/documents/edl-v10.php.
 */

You can list yourself as having copyright.